setup.py

```diff
@@ -1,6 +1,8 @@
 from distutils.core import setup
 from setuptools import find_packages
+from setuptools.command.install import install
 from sistr.version import __version__
+from sistr.sistr_cmd import setup_sistr_dbs
```

Comment:
I'm wondering if you can move this to the `run()` function below, so that SISTR (and dependencies) aren't imported before installation. Here's the error I get when I try to install from a clean conda environment:

```
Traceback (most recent call last):
  File "setup.py", line 5, in <module>
    from sistr.sistr_cmd import setup_sistr_dbs
  File "/home/CSCScience.ca/apetkau/workspace/sistr_cmd_2/sistr/sistr_cmd.py", line 14, in <module>
    from sistr.src.blast_wrapper import BlastRunner
  File "/home/CSCScience.ca/apetkau/workspace/sistr_cmd_2/sistr/src/blast_wrapper/__init__.py", line 7, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
```

Reply:
It turned out that `sistr_cmd.py` has a number of imports (including `pandas`) that are not available during install from scratch. This issue was solved by installing database after the package install as at that time all dependencies are available.
Installation via `pip install .` route will not initialize SISTR databases and ignore custom install, but the databases will be initialized automatically upon first run on input (i.e. genome) and `sistr -h` command would not give any missing package resources error (as is the case with `python setup.py  install` route).

Reply:
Sounds good, thanks.